src/client/components/AddReview.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useContext } from "react";
+import Header from "./Header";
+import Footer from "./Footer";
+import DataContext from "./DataContext";
+
+export default function AddReview() {
+  const { upcomingMeals } = useContext(DataContext);
+  const [inputState, setInputstate] = useState(false);
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const [mealId, setMealId] = useState("");
+  const [stars, setStars] = useState("");
+  function newReview() {
```

Comment:
Try to name functions as verbs or action. "newReview" is a noun, how about "addNewReview"?